relay-dynamic-config/src/feature.rs

```diff
@@ -6,67 +6,97 @@ use serde::{Deserialize, Serialize};
 #[derive(Clone, Debug, Eq, PartialEq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
 pub enum Feature {
     /// Enables ingestion of Session Replays (Replay Recordings and Replay Events).
+    ///
+    /// Serialized as `organizations:session-replay`.
```

Comment:
Would be nice if rustdoc could show this automatically, but I could not find a setting for it.

A proc macro could probably automate this, but I did not want to spend time on that now.